### PR TITLE
Updated screenshot for issue #651

### PR DESCRIPTION
**Pull request by aditya-sarkar441**
I have updated the zulip-term screenshot for zulip-terminal issue #651. Kindly check.

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ If you encountered any issues above or have any queries, feel free to ask for he
 I would like you to try out a few of our keyboard shortcuts and send a couple message to help you get familiar with zulip terminal.
 
 #### What do you see?
-![An image of working Zulip Terminal](https://camo.githubusercontent.com/d4efe459c212e64ae7c841e5dda288a14837bf1e/68747470733a2f2f64726976652e676f6f676c652e636f6d2f75633f6578706f72743d766965772669643d31746163325053535f47625177587a34476e7a66636f5f68696e5776464f327a61)
+![An image of working Zulip Terminal](https://github.com/aditya-sarkar441/Screenshots/blob/master/Screenshot%202%202020-05-15%20at%209.31.03%20PM.png)
```
[the remark this answers is not in the document]
Comment (aditya-sarkar441) [reply docs/getting-started.md:11]: Ok, I can use camo from GitHub for that.

File: README.md [hunk 2]
```diff
@@ -16,6 +16,9 @@ An interactive terminal interface for [Zulip](https://zulipchat.com).
 - OSX
 - WSL (On Windows)
 
+## What do you see
```
Comment (preetmishra) [inline README.md:19]: For the `README`, you might want to use a heading that's more appropriate than 'What do you see'.

File: docs/getting-started.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ If you encountered any issues above or have any queries, feel free to ask for he
 I would like you to try out a few of our keyboard shortcuts and send a couple message to help you get familiar with zulip terminal.
 
 #### What do you see?
-![An image of working Zulip Terminal](https://github.com/aditya-sarkar441/Screenshots/blob/master/Screenshot%202%202020-05-15%20at%209.31.03%20PM.png)
+<img src='https://user-images.githubusercontent.com/56690786/82113602-e27c0300-9774-11ea-9631-b5e6ee204a0c.png'>
```
Comment (preetmishra) [inline docs/getting-started.md:11]: Is there any particular reason why you decided to go with HTML rather the existing markdown syntax?

Comment (aditya-sarkar441) [reply docs/getting-started.md:11]: No such specific reasons. Both are working fine.

File: README.md (hunk 2 above)
Comment (aditya-sarkar441) [reply README.md:19]: Sure, I will change the heading.

Comment (aditya-sarkar441) [reply README.md:19]: I wasn't able to understand what @neiljp mentioned regarding the announce stream. I will be glad if you can elaborate a little bit.

File: docs/getting-started.md (hunk 3 above)
Comment (preetmishra) [reply docs/getting-started.md:11]: For a cleaner diff, it would be better to replace the link in the existing markdown.

Comment (aditya-sarkar441) [reply docs/getting-started.md:11]: Sure, I'll do it.

**Comment by aditya-sarkar441** [comment]
@preetmishra I hope it's fine now.

File: docs/getting-started.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (neiljp) [reply docs/getting-started.md]: I'm not sure if this is from camo, but it seems to work :+1:

File: README.md (hunk 2 above)
Comment (neiljp) [reply README.md]: @aditya-sarkar441 It seems that you resolved the #announce issue, based on the last screenshot.

File: README.md [hunk 4]
```diff
@@ -16,6 +16,9 @@ An interactive terminal interface for [Zulip](https://zulipchat.com).
 - OSX
 - WSL (On Windows)
 
+## Image of Zulip-Terminal
+![image](https://user-images.githubusercontent.com/56690786/82187341-e4c39600-9909-11ea-9037-165f24d2691e.png)
```
Comment (neiljp) [inline README.md:20]: Thanks for adding this here too :+1:

I think we likely want to place this above the `Supported Platforms` to get this more visibility, and we don't really need a heading for it?

File: docs/getting-started.md [hunk 5]
```diff
@@ -8,7 +8,7 @@ If you encountered any issues above or have any queries, feel free to ask for he
 I would like you to try out a few of our keyboard shortcuts and send a couple message to help you get familiar with zulip terminal.
 
 #### What do you see?
-![An image of working Zulip Terminal](https://camo.githubusercontent.com/d4efe459c212e64ae7c841e5dda288a14837bf1e/68747470733a2f2f64726976652e676f6f676c652e636f6d2f75633f6578706f72743d766965772669643d31746163325053535f47625177587a34476e7a66636f5f68696e5776464f327a61)
+![image]('https://user-images.githubusercontent.com/56690786/82187341-e4c39600-9909-11ea-9037-165f24d2691e.png')
```
Comment (neiljp) [inline docs/getting-started.md:11]: This doesn't seem to render; is the link right?

Comment (aditya-sarkar441) [reply docs/getting-started.md:11]: Link is working fine. Kindly check.

File: README.md (hunk 4 above)
Comment (aditya-sarkar441) [reply README.md:20]: Sure

**Pull request closed without merging**